libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -20,14 +20,17 @@
  * Static analyzer and checker.
  */
 
-#include <libsolidity/analysis/StaticAnalyzer.h>
 #include <memory>
+#include <libsolidity/analysis/StaticAnalyzer.h>
 #include <libsolidity/ast/AST.h>
+#include <libsolidity/inlineasm/AsmAnalysis.h>
+#include <libsolidity/inlineasm/AsmAnalysisInfo.h>
+#include <libsolidity/inlineasm/AsmData.h>
 
 using namespace std;
 using namespace dev;
 using namespace dev::solidity;
-
+using namespace dev::solidity::assembly;
```

Comment:
I'm not sure if we should do this. There are some identifiers that exist both in `dev::solidity` and `dev::assembly`.